web/src/Layout.jsx

```diff
@@ -25,6 +25,7 @@ import "./layout.scss";
 import logo from "./assets/suse-horizontal-logo.svg";
 
 import About from "./About";
+import Ip from "./Target";
```

Comment:
The name "ip" is OK for a local variable, but I think it is too ambiguous here. What about "TargetMachineAddress"